### PR TITLE
models_link

### PR DESCRIPTION
**Pull request by mohamedsayed18**

### CODE REVIEW

**Comment by chapulina** [comment]
Thanks for opening the PR, @mohamedsayed18 ! Mind signing the commit for the DCO check?

**Comment by mohamedsayed18** [comment]
signed the commit

File: examples/worlds/fuel.sdf [hunk 1]
```diff
@@ -121,12 +121,12 @@
...
+      <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Double pendulum with base</uri>
     </include>
 
     <include>
       <pose>1 0 0 0 0 0</pose>
-      <uri>https://fuel.ignitionrobotics.org/1.0/nate/models/my_Backpack</uri>
+      <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Backpack</uri>
     </include>
 
     <include>
```
Comment (mohamedsayed18) [inline examples/worlds/fuel.sdf:132]: should we change this line also

**Comment by mohamedsayed18** [comment]
Should I also edit line 134?
`<uri>https://fuel.ignitionrobotics.org/1.0/chapulina/models/Gazebo - relative paths</uri>`

**Comment by chapulina** [comment]
> Should I also edit line 134?

Good call, I just copied that model from my account to OpenRobotics, so you can use this one now:

https://app.ignitionrobotics.org/OpenRobotics/fuel/models/Gazebo%20-%20relative%20paths

**Comment by mohamedsayed18** [comment]
> Good call, I just copied that model from my account to OpenRobotics, so you can use this one now:
Done

**Pull request merged**